python/pycylon/pycylon/data/table.pyx

```diff
@@ -1197,6 +1197,56 @@ cdef class Table:
             final_table = Table.from_list(self.context, col_names, filtered_all_data)
             return final_table
 
+    cdef _get_head_response(self, offset, length):
+        cdef shared_ptr[CTable] output
+        cdef CStatus status
+        status = Head(self.table_shd_ptr, length, &output)
```

Comment:
It is better to do as follows,

```suggestion
        cdef:
            shared_ptr[CTable] output
            CStatus status
```

Make sure the tabs are okay for indentation. Don't commit the suggestion, I am not sure if this reflects accurately.

Reply:
I will reflect the suggestions